### PR TITLE
fix: dashboard updates for sumac

### PR DESCRIPTION
**Pull request by saraburns1**
1. Upgrade dbt version
2. Remove course comparison dashboard from instructor tab
3. Change y axis to Adaptive Formatting (prevents duplicate numbers)
4. Fix 'active within past 7 days' metric (was duplicating count)
5. Use 'video count' instead of 'video id' for course comparison dash

### CODE REVIEW

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml [hunk 1]
```diff
@@ -47,31 +42,56 @@ params:
...
+      python_date_format: null
+      type: UInt64
+      type_generic: 0
+      verbose_name: Active Within Last 7 Days
+      warning_markdown: null
+    datasourceWarning: false
+    expressionType: SIMPLE
+    hasCustomLabel: true
+    label: Active Within Last 7 Days
+    optionName: metric_9ysvoht7z9_7zzqrfsnybt
```
Comment (Ian2012) [inline tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml:78]: Metrics should be added to the datasets, instead of using the charts metrics, (translations)

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml [hunk 2]
```diff
@@ -178,6 +264,6 @@ query_context:
   result_format: json
   result_type: full
 slice_name: Course Information
-uuid: fa249dda-78da-4ccc-9ef3-39177e6aae0c
+uuid: f909e03a-06a9-4d04-9ea0-243a576093d4
```
Comment (Ian2012) [inline tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml:267]: Changing those assets UUIDs can cause some problems. Please revert this change and all the references to this new UUID

**Review by Ian2012 — CHANGES_REQUESTED** [review]
Just small changes before approving

**Review by Ian2012 — APPROVED** [review]

**Review by bmtcril — APPROVED** [review]
